ethereum/chain.py

```diff
@@ -345,7 +345,7 @@ def add_transaction(self, transaction):
         log.debug('add tx', num_txs=self.num_transactions(), tx=transaction, on=head_candidate)
         if self.head_candidate.includes_transaction(transaction.hash):
             log.debug('known tx')
-            return
+            return False
```

Comment:
`return None` here could be intentional, to distinguish tx already included and tx is invalid.

Can we keep the patch a minimum?